proteus/mprans/BoundaryConditions.py

```diff
@@ -169,18 +169,21 @@ def setAtmosphere(self, orientation=None, vof_air=1.):
             orientation = self._b_or
         self.reset()
         self.p_dirichlet.setConstantBC(0.)
-        if self._b_or[0] == 1. or self._b_or[0] == -1.:
-            self.u_dirichlet.setConstantBC(0.)
-        else:
-            self.u_dirichlet.resetBC()
-        if self._b_or[1] == 1. or self._b_or[1] == -1.:
-            self.v_dirichlet.setConstantBC(0.)
-        else:
-            self.v_dirichlet.resetBC()
-        if self._b_or[2] == 1. or self._b_or[2] == -1.:
-            self.w_dirichlet.setConstantBC(0.)
-        else:
-            self.w_dirichlet.resetBC()
+        self.u_dirichlet.setConstantBC(0.)
+        self.v_dirichlet.setConstantBC(0.)
+        self.w_dirichlet.setConstantBC(0.)
+        # if self._b_or[0] == 1. or self._b_or[0] == -1.:
+        #     self.u_dirichlet.setConstantBC(0.)
```

Comment:
I was in a hurry and didn't have time to fix these commented out sections, but these are where you want to turn on  the diffusive flux if the (and leave the Dirichlet conditions alone).